### DISCUSSION TITLE
regarding changing theme files

### DISCUSSION THREAD

**Discussion by beartester34** [GetPublii/Publii#1729] [Help] [answered]
Hello everyone.

I have a query regarding the changes made to theme files.

Suppose, I want to make changes in the theme files of Publii. I did it by changing the theme files which I got by exporting the website as a zip archive, by syncing the website.

However, if i again sync the website, the changes that I have made in the theme files, will be overwritten and hence the whole exercise of changing the theme files will go futile.

I then came across a tutorial, https://getpublii.com/docs/overriding-publii-theme.html
Where we can use over ride to make sure that the changes made to the theme files, wont be overridden.

However, what I can't get my mind around to is , how to make changes to the theme files as it has been mentioned in this tutorial.

**Comment by beartester34** [comment]
for example , I want to make changes in the files in the output directory 

theme-name/tags/index.html

so that, it doesn't get overridden when i sync my website again.

**Reply by candidexmedia** [reply]
You're not meant to override the files in the `output` folder, but the theme files in the `input`.

> Inside this directory you'll see folders for each of your Publii websites; each website can have its own overrides, so open the folder for the site you wish to change. You'll see three folders; input, output and preview; the themes folder will be in the input folder, so open this, and finally, open the themes folder; this contains the themes currently installed for your chosen website.

What kinds of changes are you making?

**Reply by beartester34** [reply]
> You're not meant to override the files in the `output` folder, but the theme files in the `input`.
> 
> > Inside this directory you'll see folders for each of your Publii websites; each website can have its own overrides, so open the folder for the site you wish to change. You'll see three folders; input, output and preview; the themes folder will be in the input folder, so open this, and finally, open the themes folder; this contains the themes currently installed for your chosen website.
> 
> What kinds of changes are you making?

Hello

Thanks for replying. I want to make some changes and If I were to understand how to make one correctly, I can maybe make other changes as well. I will give an example of a change that I want to make.

for example in this theme
https://demo.getpublii.eu/themes/q-answer/2/tags/

i want to change the wordings of 

Tags

Collection of all tags

to

Dogs

Collection of all Dogs


![1](https://github.com/user-attachments/assets/cbbbc93b-e164-4c5d-b47e-63564f779b9a)

The only way I could figure out to do it was to change the index.html file in outputfile/tags/index.html

How to go about doing this change. I hope I am making sense.

**Reply by bobmitro** [reply, marked as answer]
You can find these phrases in the language file.

More details:  https://getpublii.com/docs/translate-publii-theme-to-another-language.html

**Reply by beartester34** [reply]
> You can find these phrases in the language file.
> 
> More details: https://getpublii.com/docs/translate-publii-theme-to-another-language.html

Thanks a ton. This works perfectly.
